Firebase/Messaging/FIRMessaging.m

```diff
@@ -233,28 +240,19 @@ + (void)load {
                                  withInstanceID:[FIRInstanceID instanceID]
                                withUserDefaults:[GULUserDefaults standardUserDefaults]];
     [messaging start];
+    [messaging configureNotificationSwizzlingIfEnabled];
     return messaging;
   };
   FIRComponent *messagingProvider =
       [FIRComponent componentWithProtocol:@protocol(FIRMessagingInstanceProvider)
-                      instantiationTiming:FIRInstantiationTimingLazy
+                      instantiationTiming:FIRInstantiationTimingEagerInDefaultApp
```

Comment:
Why this change from lazy to eager?

Reply:
It mimics the previous behaviour of `configureWithApp:`, where Messaging would instantiate itself as soon as `configureWithApp:` is called.